docs/documentation/release_notes/topics/26_4_0.adoc

```diff
@@ -194,7 +194,7 @@ this by setting `spec.serviceMonitor.enabled: false`. For more details, see the
 {project_name} supports HTTP access logging to record details of incoming HTTP requests.
 While access logs are often used for debugging and traffic analysis, they are also important for security auditing and compliance monitoring.
 
-For more information, see https://www.keycloak.org/server/logging[Configuring logging].
+For more information, see https://www.keycloak.org/server/logging#_http_access_logging[Configuring logging].
```

Comment:
The automatic anchors are likely to break when someone changes the heading. A good practice is to add an explicit anchor to the section title like we do it here: 

```
[[cache-topology]]
== Topology aware data distribution
```

This prevents such problems. Please rework those targets.

Reply:
Should be good now.